WpfApp4/WpfApp4/Tools/LanguageToColorConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace WpfApp4.Tools
+{
+    public class LanguageToColorConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
```

Comment:
Where did you take DateTimeToDateConverter? Why did not you use the same approach?